### PR TITLE
Robin jira 123456

### PR DESCRIPTION
**Pull request by RobinBreyl**
### Description of Change

- things to do in Essen
This fixes #25 

### Test results

- Tested a few years ago  :smiley:

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

**Comment by rh851** [comment]
Comment in issue should be added to file.

File: Robin_Test_01.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Things to do 
+## in Essen
+There are many things to do in Essen
+### Nature
+- Lake Baldeney
```
Comment (RobinBreyl) [inline Robin_Test_01.md:5]: What about lake Kemnade in Bochum?

**Comment by rh851** [comment]
> This fixes #25
Meintest Du #15?

File: Robin_Test_01.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Things to do 
+## in Essen
+There are many things to do in Essen
+### Nature
```
Comment (flogat) [inline Robin_Test_01.md:4]: What about Best Worscht?

**Review by githubteacher — APPROVED** [review]
Great! Thank you! Now I have some things to do in Essen this weekend. 😄

**Review by flogat — APPROVED** [review]

**Comment by rh851** [comment]
Branch reopened

**Pull request merged**